controllers/operatorconfigmap_controller.go

```diff
@@ -591,3 +598,62 @@ func labelClientOperatorSubscription(c *OperatorConfigMapReconciler) error {
 	c.log.Info("successfully labelled ocs-client-operator subscription")
 	return nil
 }
+
+func (c *OperatorConfigMapReconciler) reconcileSubscription() error {
+
+	storageClients := &v1alpha1.StorageClientList{}
+	if err := c.list(storageClients); err != nil {
+		return fmt.Errorf("failed to list storageclients: %v", err)
+	}
+
+	var desiredChannel string
+	for idx := range storageClients.Items {
+		// empty if annotation doesn't exist or else gets desired channel
+		channel := storageClients.Items[idx].GetAnnotations()[utils.DesiredSubscriptionChannelAnnotationKey]
```

Comment:
This is a long line, can you please split it?
```suggestion
		channel := storageClients.
			Items[idx].
			GetAnnotations()[utils.DesiredSubscriptionChannelAnnotationKey]
```

Reply:
done.